stackslib/src/clarity_vm/clarity.rs

```diff
@@ -2471,6 +2610,10 @@ mod tests {
                 u32::MAX
             }
 
+            fn get_v3_unlock_height(&self) -> u32 {
```

Comment:
Is there a purpose to keeping these unlock height and activation height functions separated into v3/v1 and pox_3/pox_4? they always seem to be the same in which case it makes more sense to keep just a single "get_unlock_height" and a "get_pox_activation_height" ?